CHANGES.rst

```diff
@@ -132,6 +132,10 @@ Release History
   `#671 <https://github.com/nengo/nengo/pull/671>`_)
 - Fix for SPA actions that route to a module with multiple inputs.
   (`#714 <https://github.com/nengo/nengo/pull/714>`_)
+- Corrected the `'rmses'` values in `BuiltConnection.solver_info` when using
+  `NNls` and `Nnl2sL2` solvers. Previous values were for the Gram matrix, and
```

Comment:
I would cut this entry after the first sentence, and link to the PR.

The changelog isn't designed to replace what already happens in Github and in commit messages, it's to summarize what's happened as briefly as possible; if people want to know more, then the Github stuff is there.

Also, note that this file is reStructuredText, not markdown, so you need double backticks to render the text as code.